io.kapsa.drive.json

```diff
@@ -0,0 +1,338 @@
+          "sha256": "b9cc4a2acd74509435892823607d966d424bd9ad5d0b00938f27240a1bfa878a"
+        }
+      ],
+      "cleanup": [
+        "/include",
+        "/libexec",
+        "/lib/pkgconfig",
+        "/lib/*.la",
+        "/share/doc",
+        "/share/libdbusmenu",
+        "/share/gtk-doc",
+        "/share/gir-1.0"
+      ]
+    },
+    {
+      "name": "ayatana-ido",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/AyatanaIndicators/ayatana-ido.git",
+          "tag": "0.9.1"
+        }
+      ]
+    },
+    {
+      "name": "libayatana-indicator",
+      "buildsystem": "cmake-ninja",
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/AyatanaIndicators/libayatana-indicator.git",
+          "tag": "0.9.1"
+        }
+      ]
+    },
+    {
+      "name": "libayatana-appindicator",
+      "buildsystem": "cmake-ninja",
+      "config-opts": [
+        "-DENABLE_BINDINGS_MONO=NO",
+        "-DENABLE_BINDINGS_VALA=NO"
+      ],
+      "sources": [
+        {
+          "type": "git",
+          "url": "https://github.com/AyatanaIndicators/libayatana-appindicator.git",
+          "tag": "0.5.90"
+        }
+      ]
+    },
+    {
+      "name": "libjson",
+      "buildsystem": "meson",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/open-source-parsers/jsoncpp/archive/refs/tags/1.9.5.zip",
+          "sha256": "a074e1b38083484e8e07789fd683599d19da8bb960959c83751cd0284bdf2043"
+        }
+      ]
+    },
+    {
+      "name": "libsecret",
+      "buildsystem": "meson",
+      "config-opts": [
+        "-Dmanpage=false",
+        "-Dvapi=false",
+        "-Dgtk_doc=false",
+        "-Dintrospection=false"
+      ],
+      "cleanup": [
+        "/bin",
+        "/include",
+        "/lib/pkgconfig",
+        "/share/man"
+      ],
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://download.gnome.org/sources/libsecret/0.20/libsecret-0.20.5.tar.xz",
+          "sha256": "3fb3ce340fcd7db54d87c893e69bfc2b1f6e4d4b279065ffe66dac9f0fd12b4d"
+        }
+      ]
+    },
+    {
+      "name": "libsodium",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/jedisct1/libsodium/archive/1.0.18-RELEASE.tar.gz",
+          "sha256": "b7292dd1da67a049c8e78415cd498ec138d194cfdb302e716b08d26b80fecc10",
+          "x-checker-data": {
+            "type": "anitya",
+            "project-id": 1728,
+            "url-template": "https://github.com/jedisct1/libsodium/archive/$version.tar.gz"
+          }
+        }
+      ],
+      "post-install": [
+        "install -Dm644 LICENSE ${FLATPAK_DEST}/share/licenses/libsodium/LICENSE"
+      ]
+    },
+    {
+      "name": "s3drive",
+      "buildsystem": "simple",
+      "only-arches": [
+        "x86_64"
```

Comment:
this doesn't prevent an aarch64 flatpak to be built.

Reply:
Sorry, I am not exactly experienced with that.
What would be your suggestion? 
As such we haven't tested it on ARM and we would like to limit platforms to standard 32 / 64-bit PC.

Reply:
from the documentation you ackowledge having read:
https://docs.flathub.org/docs/for-app-authors/maintanance/#limiting-the-set-of-architectures-to-build-on